src/Mvc/Mvc.Api.Analyzers/test/Infrastructure/MvcTestSource.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.Analyzer.Testing;
+using Microsoft.AspNetCore.Testing;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public static class MvcTestSource
```

Comment:
This file looks like a duplicate of src/Mvc/Mvc.Analyzers/test/Infrastructure/MvcTestSource.cs. Intentional?

Reply:
Yeah. Because we were sharing the same file between two assemblies and using assemblyname to figure out the folder structure. Since we can't do that anymore, I had to duplicate